runtime/src/main/java/org/corfudb/runtime/collections/DeltaMessage.java

```diff
@@ -0,0 +1,49 @@
+package org.corfudb.runtime.collections;
+
+import lombok.Getter;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class DeltaMessage {
```

Comment:
Aren't we using RoutingTableEntryMsg as the message type? When would DeltaMessage be used?

Reply:
Yes it should be RoutingTableEntryMsg.